packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -1288,24 +1290,28 @@ class ListView extends BoxScrollView {
 /// Here are two brief snippets showing a [GridView] and its equivalent using
```

Comment:
Let's split this description between the two samples. The second sample can mention it creates the same visual layout as this sample above.